src/views/Header/HeaderBackButtonWrapper.js

```diff
@@ -0,0 +1,72 @@
+/* @flow */
+
+import * as React from 'react';
+import { I18nManager, View, Platform, StyleSheet } from 'react-native';
+
+import TouchableItem from '../TouchableItem';
+
+type Props = {
+  onPress?: () => void,
+  pressColorAndroid?: string,
+  accessibilityLabel?: string,
+  children: React.ChildrenArray<*>,
+};
+
+type DefaultProps = {
+  pressColorAndroid: string,
+  accessibilityLabel: string,
+};
+
+class HeaderBackButtonWrapper extends React.PureComponent<DefaultProps, Props> {
```

Comment:
Oh, `DefaultProps` shouldn't be here! Give [this](https://medium.com/flow-type/even-better-support-for-react-in-flow-25b0a3485627) a read. That's definitely your problem. Make sure you're using modern React types throughout this PR!